typescript-selenium/utils/DriverHelper.ts

```diff
@@ -0,0 +1,274 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { Driver } from "../driver/Driver";
+import { inject, injectable } from "inversify";
+import { TYPES } from "../inversify.types";
+import { error } from 'selenium-webdriver';
+import 'reflect-metadata';
+import { WebElementPromise, ThenableWebDriver, By, promise, until, WebElement, WebElementCondition } from "selenium-webdriver";
```

Comment:
Most of these imports are not used. I would like to see `tslint` introduced (you can reuse config from dashboard).